entrypoint/set-user.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eu
+
+# run the docker imaage with: -e LOCAL_USER_ID=`id -u $USER`
+# source: https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+user=user
+USER_ID=${LOCAL_USER_ID:-9001}
```

Comment:
Why not use the UID/GID of the user who runs `docker`? That way the behavior matches non-dockerized workflows.